.github/workflows/release-drafter.yml

```
@@ -0,0 +1,16 @@
+name: Release Drafter
+
+on:
+  push:
+   
+    branches:
+      - main 
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run Release Drafter
+        uses: release-drafter/release-drafter@v5
+        
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

The `secrets.GITHUB_TOKEN` secret is available for workflows out of the box. But you would need write permissions:

```suggestion
name: Release Drafter

on:
  push:
    branches:
      - main

permissions:
  contents: write

jobs:
  update_release_draft:
    runs-on: ubuntu-latest
    steps:
      - name: Run Release Drafter
        uses: release-drafter/release-drafter@v5
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```